NuGet.config

```diff
@@ -3,13 +3,10 @@
 <configuration>
   <packageSources>
     <clear />
-    <add key="myget-fxlab" value="https://dotnet.myget.org/F/dotnet-corefxlab/api/v3/index.json" />
     <add key="arcade" value="https://dotnetfeed.blob.core.windows.net/dotnet-tools-internal/index.json" />
-    <add key="dotnet-core" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
-    <add key="dotnet-coreclr" value="https://dotnetfeed.blob.core.windows.net/dotnet-coreclr/index.json" />
-    <add key="myget-core" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />
+    <add key="dotnet3" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
+    <add key="dotnet3-transport" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
+    <add key="dotnet-tools" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json" />
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
-    <add key="dotnet-wd" value="https://dotnetfeed.blob.core.windows.net/dotnet-windowsdesktop/index.json" />
```

Comment:
We need this feed, these contains WPF/WinForms transport packages.

Reply:
These are also now automatically pushed to an azdo feed, and so should be able to be removed.

Reply:
To be clear, the goal is to get down to:
- nuget.org
- Azdo feeds

I'm speculatively poking at some of this as I update these feeds now to get a head start.

Reply:
Ah ok, sorry I was misunderstanding before and I saw the build error looking for WinForms.

I'm making similar changes in our test repo, so I'll wait on this for a final version before moving it there.

Reply:
What is the mapping between these feeds and the AzDo transport feeds? For eg, which AzDo feed gets packages from the dotnet-windowsdesktop feed?

I like the idea of human understandable NuGet.config...

Reply:
- All dotnet 3 transport packages go to dotnet3-transport
- All dotnet 3 non-transport (shipping) packages go to dotnet3
- All dotnet 3.1 transport packages go to dotnet3.1-transport
- All dotnet 3.1 non-transport (shipping) packages go to dotnet3.1
- All dotnet 5 transport packages go to dotnet5-transport
- All dotnet 5 non-transport (shipping) packages go to dotnet5
- Arcade's tools go to dotnet-tools

There's no segregation between the various repos. That was largely a result of the scalability issues with the blob feed push.